app/helpers/admin/tabbed_nav_helper.rb

```diff
@@ -77,7 +77,7 @@ def policy_group_nav_items(group, current_path)
       {
         label: "Group",
         href: edit_admin_policy_group_path(group),
-        current: current_path == edit_admin_policy_group_path(group),
+        current: current_path == edit_admin_policy_group_path(group) || current_path == admin_policy_group_path(group),
```

Comment:
Not sure I fully understand this. Would be good to discuss how the tabbed_nav works.

Reply:
The tabbed nav takes an array of hashes with a  label, href and current (functions a bit like selected or checked for radios/selects/checkboxes).

When current == true it gives the  tab the selected styling (blue underline)

<img width="364" alt="image" src="https://user-images.githubusercontent.com/42515961/231088765-516e887d-4ab6-45f4-9122-718cebf9a79b.png">

This is just a helper method that returns those array of hashes. I think there's probably a subtle bug at play here which goes further than just this tab.

When an edit page is passed in as the current path, then an unsuccessful save occurs, the update action in the controller re-renders the edit view. Due to this the current path is the update actions path, not the edit path. i.e.

```
/government/admin/policy-groups/1330557
```

instead of
```
/government/admin/policy-groups/1330557/edit
```

Due to this the tab was re-rendering with no tabs showing the selected styling

Reply:
Thanks @davidgisbey, that makes sense to me. It was actually the 'send' method in tabbed_nav_helper that was throwing me. But understand now.